ui/pages/onboarding-flow/secure-your-wallet/secure-your-wallet.js

```diff
@@ -121,56 +118,53 @@ export default function SecureYourWallet() {
           {t('seedPhraseIntroRecommendedButtonCopy')}
         </Button>
       </Box>
-      <Box marginBottom={4} textAlign={TEXT_ALIGN.CENTER}>
-        <Typography
-          tag="span"
-          variant={TYPOGRAPHY.H4}
-          fontWeight={FONT_WEIGHT.BOLD}
-          boxProps={{ display: 'block' }}
-        >
-          {t('seedPhraseIntroSidebarTitleOne')}
-        </Typography>
-        <Typography tag="span" variant={TYPOGRAPHY.H4}>
-          {t('seedPhraseIntroSidebarCopyOne')}
-        </Typography>
-      </Box>
-      <Box marginBottom={4} textAlign={TEXT_ALIGN.CENTER}>
-        <Typography
-          tag="span"
-          variant={TYPOGRAPHY.H4}
-          fontWeight={FONT_WEIGHT.BOLD}
-          boxProps={{ display: 'block' }}
-        >
-          {t('seedPhraseIntroSidebarTitleTwo')}
-        </Typography>
-        <ul className="secure-your-wallet__list">
-          <li>{t('seedPhraseIntroSidebarBulletOne')}</li>
-          <li>{t('seedPhraseIntroSidebarBulletTwo')}</li>
-          <li>{t('seedPhraseIntroSidebarBulletThree')}</li>
-          <li>{t('seedPhraseIntroSidebarBulletFour')}</li>
-        </ul>
-      </Box>
-      <Box marginBottom={6} textAlign={TEXT_ALIGN.CENTER}>
-        <Typography
-          tag="span"
-          variant={TYPOGRAPHY.H4}
-          fontWeight={FONT_WEIGHT.BOLD}
-          boxProps={{ display: 'block' }}
-        >
-          {t('seedPhraseIntroSidebarTitleThree')}
-        </Typography>
-        <Typography tag="span" variant={TYPOGRAPHY.H4}>
-          {t('seedPhraseIntroSidebarCopyTwo')}
-        </Typography>
-      </Box>
-      <Box
-        className="secure-your-wallet__highlighted"
-        marginBottom={2}
-        textAlign={TEXT_ALIGN.CENTER}
-      >
-        <Typography tag="span" variant={TYPOGRAPHY.H4}>
-          {t('seedPhraseIntroSidebarCopyThree')}
-        </Typography>
+      <Box>
+        <Box marginBottom={4}>
+          <Typography
+            tag="span"
+            variant={TYPOGRAPHY.H4}
+            fontWeight={FONT_WEIGHT.BOLD}
+            boxProps={{ display: 'block' }}
```

Comment:
If we're going to `display: block` this, we likely don't need a `tag="span"`.

Reply:
@darkwing I changed `tag="span"` to `tag="p"` as @brad-decker suggested.
Fixed in: 58720e7d4f52a70703f7d7f6502b31dcbe7493b3